src/physics/systems.rs

```diff
@@ -93,35 +93,32 @@ pub fn sync_transform_system(
     bodies: ResMut<RigidBodySet>,
     scale: Res<RapierPhysicsScale>,
     rigid_body: &RigidBodyHandleComponent,
-    mut translation: Mut<Translation>,
-    mut rotation: Mut<Rotation>,
+    mut transform: Mut<Transform>,
 ) {
     if let Some(rb) = bodies.get(rigid_body.handle()) {
         let pos = rb.position;
 
         #[cfg(feature = "dim2")]
         {
             let rot = na::UnitQuaternion::new(na::Vector3::z() * pos.rotation.angle());
-
-            *translation.0.x_mut() = pos.translation.vector.x * scale.0;
-            *translation.0.y_mut() = pos.translation.vector.y * scale.0;
-            rotation.0 = Quat::from_xyzw(rot.i, rot.j, rot.k, rot.w);
+            transform.set_translation(Vec3::new(pos.translation.vector.x * scale.0, pos.translation.vector.y * scale.0, 0.0));
```

Comment:
I guessed that the `z` value for 2d should be `0.0` here.

Reply:
z value is used in bevy for sprite layering, so I think rapier should not touch the value set originally by the user.